.github/workflows/testing.yaml

```diff
@@ -30,8 +30,9 @@ jobs:
       - name: "Checkout sample project repository"
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          repository: "lfreleng-actions/test-python-project"
+          repository: "modeseven-lfreleng-actions/test-python-project"
```

Comment:
You're changing this to your personal fork instead of the `lfreleng-actions/test-python-project`.